tools/src/main/scala/scala/scalanative/build/NativeConfig.scala

```diff
@@ -192,8 +192,8 @@ object NativeConfig {
   /** Default empty config object where all of the fields are left blank. */
   def empty: NativeConfig =
     Impl(
-      clang = Paths.get(""),
-      clangPP = Paths.get(""),
+      clang = Paths.get("clang"),
+      clangPP = Paths.get("clang++"),
```

Comment:
These seem ok. Just wondering the reasoning.

Reply:
When testing this PR I've observed some failures, when detecting clang version. That happened due to some quite unexpected reasons caused by an unset clang setting. Because of that, I've gotten an error when trying to run "/home/<path-to-project>/2.13" which is a directory. It look quite strange to me, so instead I've decided to set clang on path by default. 
However, current setting is incorrect - when using clang we would try to get absolute path, which in that case would be evaluated to `currentDirectory/clang` which also is bad. I would revert this change because of that.